lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -2743,11 +2743,11 @@ if (!sourceList)
 
         hr = this->GetDebugContext()->RundownSourcesAndReparse(shouldPerformSourceRundown, /*shouldReparseFunctions*/ true);
 
-        // Debugger attach/detach failure is catastrophic, take down the process
-        DEBUGGER_ATTACHDETACH_FATAL_ERROR_IF_FAILED(hr);
-
         if (!this->IsClosed())
```

Comment:
Oh - we are not proceeding. I go confused that there is something after this 'if'.